packages/console/src/pages/SignInExperience/components/ConnectorsTransfer.tsx

```diff
@@ -29,15 +33,32 @@ const ConnectorsTransfer = ({ value, onChange }: Props) => {
   }
 
   const datasource = data
-    ? data.map(({ target, name, enabled }) => ({
-        value: target,
-        title: (
-          <UnnamedTrans
-            resource={name}
-            className={conditionalString(!enabled && styles.disabled)}
-          />
-        ),
-      }))
+    ? data
+        .filter(({ type }) => type === ConnectorType.Social)
+        .map(({ target, name, enabled, connectors, logo }) => ({
+          value: target,
+          title: (
+            <div className={styles.title}>
+              <div className={styles.logo}>
+                <img src={logo} alt={target} />
+              </div>
+              <UnnamedTrans
+                resource={name}
+                className={conditionalString(!enabled && styles.disabled)}
+              />
+              {connectors.length > 1 &&
+                connectors
+                  .filter(({ enabled }) => enabled)
+                  .map(({ platform }) => (
+                    <div key={platform} className={styles.icon}>
+                      {platform === ConnectorPlatform.Web && <Web />}
+                      {platform === ConnectorPlatform.Native && <Native />}
+                      {platform === ConnectorPlatform.Universal && <Universal />}
+                    </div>
```

Comment:
It seems I've seen this in another PR. So is this a duplication of code?
If yes, then consider extracting it into a component, maybe?

Reply:
LOG-2574